### PR TITLE
Add DAG for LookML generation

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: dags/probe_scraper.py [hunk 1]
```diff
@@ -91,3 +91,31 @@
...
+        env_vars={
+            "GIT_SSH_KEY_BASE64": "{{ var.values.looker_repos_secret_git_ssh_key_b64 }}",
+            "HUB_REPO_URL": "git@github.com:mozilla/looker-hub.git",
+            "HUB_BRANCH_SOURCE": "base",
+            "HUB_BRANCH_PUBLISH": "main-stage",
+            "SPOKE_REPO_URL": "git@github.com:mozilla/looker-spoke-default.git",
+            "SPOKE_BRANCH_PUBLISH": "main-stage",
+            "GCLOUD_SERVICE_KEY": "{{ var.values.looker_read_gcloud_service_key }}",
+            "LOOKER_INSTANCE_URI": "https://mozillastaging.cloud.looker.com",
+            "LOOKER_API_CLIENT_ID": "{{ var.values.looker_api_client_id }}",
```
Comment (jasonthomas) [inline dags/probe_scraper.py:116]:
```suggestion
            "LOOKER_API_CLIENT_ID": "{{ var.values.looker_api_client_id_staging }}",
```

File: dags/probe_scraper.py [hunk 2]
```diff
@@ -91,3 +91,31 @@
...
+            "LOOKER_API_CLIENT_SECRET": "{{ var.values.looker_api_client_secret }}",
```
Comment (jasonthomas) [inline dags/probe_scraper.py:117]:
```suggestion
            "LOOKER_API_CLIENT_SECRET": "{{ var.values.looker_api_client_secret_staging }}",
```

File: dags/probe_scraper.py [hunk 3]
```diff
@@ -91,3 +91,31 @@
...
+        gcp_conn_id="google_cloud_derived_datasets",
+        dag=dag,
+        env_vars={
+            "GIT_SSH_KEY_BASE64": "{{ var.values.looker_repos_secret_git_ssh_key_b64 }}",
+            "HUB_REPO_URL": "git@github.com:mozilla/looker-hub.git",
+            "HUB_BRANCH_SOURCE": "base",
+            "HUB_BRANCH_PUBLISH": "main-stage",
+            "SPOKE_REPO_URL": "git@github.com:mozilla/looker-spoke-default.git",
+            "SPOKE_BRANCH_PUBLISH": "main-stage",
+            "GCLOUD_SERVICE_KEY": "{{ var.values.looker_read_gcloud_service_key }}",
```
Comment (fbertsch) [inline dags/probe_scraper.py:114]:
```suggestion
```

File: dags/probe_scraper.py [hunk 4]
```diff
@@ -91,3 +91,30 @@
...
+    delay_python_task = PythonOperator(
+        task_id="wait_for_30_minutes",
+        dag=dag,
+        python_callable=lambda: time.sleep(60 * 30))
+
+    lookml_generator = GKEPodOperator(
+        email=['frank@mozilla.com', 'dataops+alerts@mozilla.com'],
+        task_id='lookml_generator',
+        name='lookml-generator-1',
+        image='mozilla/lookml-generator:latest',
```
Comment (jasonthomas) [inline dags/probe_scraper.py:104]:
```suggestion
        image='gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest',
```

File: dags/probe_scraper.py [hunk 5]
```diff
@@ -91,3 +91,30 @@
...
+        task_id="wait_for_30_minutes",
+        dag=dag,
+        python_callable=lambda: time.sleep(60 * 30))
+
+    lookml_generator = GKEPodOperator(
+        email=['frank@mozilla.com', 'dataops+alerts@mozilla.com'],
+        task_id='lookml_generator',
+        name='lookml-generator-1',
+        image='gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest',
+        gcp_conn_id="google_cloud_derived_datasets",
```
Comment (jasonthomas) [inline dags/probe_scraper.py:105]:
```suggestion
        gcp_conn_id="google_cloud_airflow_gke",
        gke_cluster_name="workloads-prod-v1",
        gke_location="us-west1",
```
Use new gke cluster.

File: dags/probe_scraper.py [hunk 6]
```diff
@@ -91,3 +91,32 @@
         dag=dag)
 
     probe_expiry_alerts.set_upstream(probe_scraper)
+
+    delay_python_task = PythonOperator(
```
Comment (jasonthomas) [inline dags/probe_scraper.py:95]: Tests are failing `NameError: name 'PythonOperator' is not defined` need to import PythonOperator

**Review by jasonthomas — APPROVED** [review]
r+ once tests pass. 

For the prod task we will need to use `looker_api_client_id_prod` and `looker_api_client_secret_prod` which have already been defined as variables in WTMO.

**Pull request merged**